src/app/shared/object-list/search-result-list-element/item-search-result/item-types/item/item-search-result-list-element.component.html

```diff
@@ -61,10 +61,10 @@
             }
           </ds-truncatable-part>
         </span>
-        @if (dso.firstMetadataValue('dc.description.abstract')) {
+        @if (firstMetadataValue('dc.description.abstract'); as abstract) {
```

Comment:
There are changes to this file that are also missing from #4737.  please compare this file with the version on `main`, as I think they should be identical.